provider/asdf/add_plugin.go

```diff
@@ -0,0 +1,69 @@
+package asdf
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/bitrise/v2/log"
+	"github.com/bitrise-io/toolprovider/provider"
+)
+
+func (a *AsdfToolProvider) InstallToolPlugin(toolName string) error {
```

Comment:
nit: the file is called `add_plugin.go` while the public method is `InstallToolPlugin`.